lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1733,25 +1733,36 @@ namespace Js
 
         JavascriptString *string;
 
+#ifndef CC_LOW_MEMORY_TARGET // we don't need this on a target with low memory
```

Comment:
I got your point on making it dynamic and scenarios we could test. This is just a side addition to this PR. There are places we can use a runtime flag and others we might have to stick with compile time definitions. 

Since this is just a reminder for future and no-one is actually using this flag, I've added an additional TODO note instead? So, I will update this part while marking others.